lib/membership/member.js

```diff
@@ -91,10 +90,14 @@ Member.prototype.evaluateUpdate = function evaluateUpdate(update) {
     if (this._isLocalOverride(update)) {
         // Override intended update. Assert aliveness!
         this.ringpop.stat('increment', 'refuted-update');
-        update = _.defaults({
+        var newIncNumber = this.Date.now();
+        update = {
+            source: this.ringpop.whoami(),
+            sourceIncarnationNumber: newIncNumber,
+            address: this.id,
             status: Member.Status.alive,
-            incarnationNumber: this.Date.now()
-        }, update);
+            incarnationNumber: newIncNumber
+        };
```

Comment:
I'm not sure I agree: a update that's causing a local override should result in a _new_ update gossiped around marking it as alive again. A node should never create a new update with fields it don't understand to be sure to not break forward compatibility.
example: let's say we add an uuid field on updates in the future. Just changing the stuff we want would result in two contradicting updates with the same uuid where nodes could easily preserve backwards compatibility by checking if an uuid is present or not.